.github/workflows/build.yml

```diff
@@ -81,20 +81,19 @@ jobs:
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
         run: mvn -B org.sonarsource.scanner.maven:sonar-maven-plugin:sonar -Dsonar.projectKey=nanalysis_nmrfx
 
-      - name: Assemble application archives
-        run: |
-          mvn -f ./nmrfx-analyst-gui/pom.xml -Dmaven.test.skip=true package
-
-      - name: Download sharepoint client
-        uses: dsaltares/fetch-gh-release-asset@master
+      - name: Trigger private installer build
+        uses: actions/github-script@v6
         with:
-          repo: "nanalysis/sharepoint-client"
-          version: "tags/v1.3"
-          file: "sharepoint-client-1.3-jar-with-dependencies.jar"
-          token: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Publish build
-        env:
-          SHAREPOINT_CLIENT_ID: ${{ secrets.SHAREPOINT_CLIENT_ID }}
-          SHAREPOINT_CLIENT_SECRET: ${{ secrets.SHAREPOINT_CLIENT_SECRET }}
-        run: bash .github/workflows/actions-publish-build.sh
\ No newline at end of file
+          github-token: ${{ secrets.PRIVATE_REPO_TOKEN }}
+          script: |
+            await github.rest.actions.createWorkflowDispatch({
+              owner: 'nanalysis',
+              repo: 'nmrfx-private',
+              workflow_id: 'build-installers.yml',
+              ref: 'NMR-5223-create-installer-on-trigger',
```

Comment:
This will need to be replaced with `master` once the private repo's branch is merged.

Reply:
Done.